architecture/infrastructure_components/overview.adoc

```diff
@@ -5,7 +5,10 @@
 :icons:
 :experimental:
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+The link:kubernetes_infrastructure.html[Kubernetes infrastructure] topic
+provides information about masters and nodes.
+
+The link:image_registry.html[Image registry] topic outlines the different
+sources of images available to OpenShift. The
+link:web_console.html[Web console] topic outlines its structure and potential
+for integration with a JVM console.
```

Comment:
FYI, we ended up removing this topic from the TOC completely as it was stub but also because it didn't seem like an overview topic at this level really added that much.